stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -295,6 +300,10 @@ end
     return A
 end
 
+Base._reverse(A::LowerTriangular, dims) = LowerTriangular(reverse((Matrix(A)); dims))
```

Comment:
This might not be correct if reversing along the dims makes the matrix no longer a lower triangular matrix.